.pre-commit-config.yaml

```diff
@@ -21,3 +21,13 @@ repos:
         name: mypy-gym-unity
         files: "gym-unity/.*"
         args: [--ignore-missing-imports, --disallow-incomplete-defs]
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.2.3
+    hooks:
+    -   id: mixed-line-ending
+        exclude: >
+            (?x)^(
+                .*cs.meta|
+                .*.css
+            )$
```

Comment:
What does this part of exclude do?

Reply:
I excluded them because I wasn't sure if we want to change them. The meta files are autogenerated by the editor, and docs/doxygen/unity.css was changed but wasn't sure if we wanted to bother with it.